R/avail_endpoints.R

```diff
@@ -1,11 +1,12 @@
-#' List all available endpoints.
+#' List all available Epidata API endpoints
 #'
-#' @description
-#' A function that prints a tibble with two columns: `Endpoint` contains the
-#' function for accessing the Delphi Epidata API endpoint along with a
-#' `Description`.
+#' Fetches a data frame of all Epidata API endpoints that can be accessed using
```

Comment:
Isn't this missing a `@description`? Otherwise seems good

Reply:
Everything after the first line/paragraph is automatically in the Description section

Reply:
For my sanity, I'd rather include a `@description` if you don't mind. I've run into gotchas where the second paragraph automatically went into the details section, and so would rather just be explicit about sections.